docs/api.md

```diff
@@ -1,18 +1,3 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Was this ran with https://github.com/bazelbuild/rules_jvm_external/blob/master/scripts/generate_docs.sh?

Reply:
It is now :) Good catch. Thank you!